ce-vm/ansible/override/cli/vagrant/www/sites/default/settings.php

```diff
@@ -51,7 +51,9 @@
 $settings['trusted_host_patterns'] = ['.*'];
 
 // @TODO: #49093 Fix Solr configuration locally and search engine on website.
-$config['search_api.server.local_solr'] = [
+// Development: Uncomment the following code to override search API solr server
```

Comment:
If we are not going to use this config later, we could remove it

Reply:
After conversation with David through Mattermost, we'll keep this block for testing purposes when we are splitting config